transformation/src/main/java/org/opensearch/migrations/VersionMatchers.java

```diff
@@ -8,6 +8,10 @@
 public class VersionMatchers {
     public static final Predicate<Version> isES_2_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 2.4"));
     public static final Predicate<Version> isES_5_X = VersionMatchers.matchesMajorVersion(Version.fromString("ES 5.6"));
+    public static final Predicate<Version> equalOrBetween_ES_5_0_and_5_4 = VersionMatchers.equalOrGreaterThanMinorVersion(Version.fromString("ES 5.0.0"))
```

Comment:
Please do not use patch version `99` that expects that a version will never have more than 100 patch release which isn't true - it can be an unbound number of patches (Even if that would be insane).  Just list these versions as the 2 digit code, e.g. `5.4`